stackslib/src/chainstate/nakamoto/tenure.rs

```diff
@@ -623,7 +649,7 @@ impl NakamotoChainState {
 
     /// Check a Nakamoto tenure transaction's validity with respect to the last-processed tenure
     /// and the sortition DB.  This validates the following fields:
-    /// * tenure_consensus_hash
+    /// * tenure_id_consensus_hash
```

Comment:
This comment is referring to the fields of the struct, so the old version is correct (unless you want to rename the struct fields).